src/domains/Lotto.js

```diff
@@ -0,0 +1,132 @@
+const MissionUtils = require('@woowacourse/mission-utils');
+const Messages = require('../constants/Messages');
+const LottoError = require('../errors/LottoError');
+
+class Lotto {
+  /** 로또가 가질 수 있는 숫자 갯수 */
+  static NUMBER_COUNT = 6;
+
+  /** 로또가 가질 수 있는 숫자의 최소값 */
+  static NUMBER_MIN = 1;
+
+  /** 로또가 가질 수 있는 숫자의 최대값 */
+  static NUMBER_MAX = 45;
+
+  /** 로또 1장의 가격 */
+  static PRICE = 1000;
+
+  /** @type {number[]} */
+  #numbers;
+
+  /**
+   * @param {number[]} numbers
+   */
+  constructor(numbers) {
+    numbers.sort((a, b) => a - b);
+    this.#numbers = numbers;
+    this.validate();
+  }
+
+  /**
+   * 무작위 번호를 가지는 로또를 생성한다.
+   *
+   * @returns {Lotto}
+   */
+  static fromRandom() {
+    const numbers = MissionUtils.Random.pickUniqueNumbersInRange(
+      Lotto.NUMBER_MIN,
+      Lotto.NUMBER_MAX,
+      Lotto.NUMBER_COUNT,
+    );
+    return new Lotto(numbers);
+  }
+
+  /**
+   * 문자열로부터 숫자를 파싱하여 로또를 생성한다.
+   *
+   * @param {string} text
+   * @returns {Lotto}
+   */
+  static fromString(text) {
+    return new Lotto(text.split(',').map(Number));
+  }
+
+  /**
+   * @param {number} money
+   * @returns {Lotto[]}
+   */
+  static buyLottos(money) {
+    if (money < Lotto.PRICE) {
+      throw new LottoError(Messages.LOTTO_BUY_AT_LEAST_ONE);
+    }
+    if (money % Lotto.PRICE !== 0) {
```

Comment:
이미 `money % Lotto.PRICE` 자체가 boolean 값을 가지기 때문에 이런식으로 간략화 할 수 있을 것 같아요!

```suggestion
    if (money % Lotto.PRICE) {
```

Reply:
불필요한 비교 연산이었던 것 같네요! 지적해주셔서 감사합니다~!!